tests/test_startup_playlist.py

```
@@ -0,0 +1,104 @@
+"""Tests for startup_playlist_id configuration and activation logic."""
+
+import asyncio
```

Flake8 (7.3.0)

[error] 'asyncio' imported but unused

(F401)